File: examples/matlab/plot_data_accel.m
%%% Plot Acceleration Data

accelData=csvread('accel.csv'); % read
accelData= accelData(2:end,:); % trim header

startTsMs = min(accelData(:,1));
tMs = accelData(:,1) - startTsMs;  % set start as 0ms
xG = accelData(:,2); % unpack
yG = accelData(:,3);
zG = accelData(:,4);

hold on;
plot(tMs,xG,'r.');
plot(tMs,yG,'b.');
plot(tMs,zG,'g.');

legend('X (g)','Y (g)','Z (g)');
title('Acceleration vs Time');
ylabel('Acceleration (g)');
xlabel('Time since Recording Start (ms)');

File: examples/python/plotting/plot_data_accel.py
import numpy as np
import matplotlib.pyplot as plt

"""
  Plot Acceleration Data
"""
import numpy as np
import matplotlib.pyplot as plt

data = np.genfromtxt('accel.csv', delimiter=',', skip_header=1, names=['t','x','y','z'])
start_ts_ms = np.amin(data['t'])
t = np.subtract(data['t'],start_ts_ms)

fig = plt.figure()

ax1 = fig.add_subplot(111) # Format figure
ax1.set_title('Acceleration vs Time')    
ax1.set_xlabel('Time since Recording Start (ms)')
ax1.set_ylabel('Acceleration (g)')

ax1.plot(t,data['x'],color='r',linestyle='None',marker='.',alpha=0.7,label='X (g)')
ax1.plot(t,data['y'],color='g',linestyle='None',marker='.',alpha=0.7,label='Y (g)')
ax1.plot(t,data['z'],color='b',linestyle='None',marker='.',alpha=0.7,label='Z (g)')

leg = ax1.legend()
plt.show() # Draw